apps/web/utils/string.tsx

```diff
@@ -42,6 +47,27 @@ export const parseStringToComponents = (html: string) => {
               );
             }
```

Comment:
This is my error but since there was only 1 case it wasn't an issue. I'm getting a warning that we need some `break;`s at the end of our cases now.

Reply:
Yeah I see, I missed that they didn't have breaks good catch. I think that's another product of my vscode env wanting to get all it's linting checks from what's in the vscode folder. In other TS repos I have it picks up on those things. 

I may make some more changes to the vscode settings.json to resolve that.